### PR TITLE
test_networking.test_l4lb is flaky

### PR DESCRIPTION
**Pull request by urbanserj**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [DCOS-56871](https://jira.mesosphere.com/browse/DCOS-56871) test_networking.test_l4lb is flaky

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_networking.py [hunk 1]
```diff
@@ -545,69 +541,56 @@ def test_app_networking_mode_with_defined_container_port(dcos_api_session, netwo
...
+            log.info('App: {}'.format(app))
+            app.deploy(dcos_api_session)
+            apps.append(app)
+        for app in apps:
+            log.info('Deploying app: {}'.format(app.id))
+            app.wait(dcos_api_session)
+        log.info('Apps are ready')
+
+        for i in range(numapps):
+            getjson('http://app-{}.marathon.l4lb.thisdcos.directory/ping'.format(i))
```
Comment (urbanserj) [inline packages/dcos-integration-test/extra/test_networking.py]: That's the actual fix.

File: packages/dcos-integration-test/extra/test_networking.py [hunk 2]
```diff
@@ -545,69 +541,56 @@ def test_app_networking_mode_with_defined_container_port(dcos_api_session, netwo
...
-    assert rs.status_code == 200
-    r = rs.json()
-    log.info('geturl {} -> {}'.format(url, r))
-    return r
-
-
 def test_l4lb(dcos_api_session):
     '''Test l4lb is load balancing between all the backends
        * create 5 apps using the same VIP
-       * get uuid from the VIP in parallel from many threads
```
Comment (urbanserj) [inline packages/dcos-integration-test/extra/test_networking.py]: Threads don't run in parallel due to GIL 😕

**Comment by urbanserj** [comment]
@mesosphere-mergebot changelog-not-required flaky test

**Comment by urbanserj** [comment]
@mesosphere-mergebot label [Ready For Review]

File: packages/dcos-integration-test/extra/test_networking.py (hunk 2 above)
Comment (jgehrcke) [reply packages/dcos-integration-test/extra/test_networking.py]: They don't run physically simultaneously, yes.

They run concurrently, though :-)

**Pull request merged**